### ISSUE TITLE
Remove namespace munging from docker provider

### ISSUE THREAD

**Issue by jberkus** [projectatomic/atomicapp#703]
Currently, AtomicApp will create fake "namespaces" for the docker provider since docker does not support these natively.  The problem with this is that it can cause atomic apps to have a fatal name collision, even though the names for the docker containers do not conflict.  This is confusing to the user and is not documented, so they cannot debug the issue.

While we could document this behavior, it seems better to just remove it, since there's no real benefit to the user.

**Comment by cdrage** [comment]
Yeah :( This was a way to find what has been deployed. Although I was the one that originally implemented it, it does muck with things.

What I *do* suggest however is adding a label to each deployment instead. Which will not only make it more non-intrusive, but work towards #642
